src/cryptonote_protocol/block_queue.cpp

```diff
@@ -29,6 +29,7 @@
 //
 // Parts of this file are originally copyright (c) 2012-2013 The Cryptonote developers
 
+#include "cryptonote_protocol_defs.h"
```

Comment:
This can be removed since the header's in `block_queue.h` now

Reply:
done